src/components/Screen.jsx

```diff
@@ -26,10 +26,10 @@ function Screen() {
       <div className="screen">
         <div className="screen-container">
           <img className="candidate-photo" src={scarlett} alt="scarlett" />
-          <button className="btnWho" type="button" onClick={() => setWho(!who)}>Qui suis-je ?</button>
-          <button className="btnPop" type="button" onClick={() => setPop(!pop)}>POP</button>
-          <button className="btnStrength" type="button" onClick={() => setStrength(!strength)}>Mes Skills</button>
-          <button className="btnWhat" type="button" onClick={() => setWhat(!what)}>What else ?</button>
+          <button className="btnWho" type="button" onClick={() => setWho(!who) || setStrength(false) || setWhat(false || setPop(false))}>Qui suis-je ?</button>
+          <button className="btnPop" type="button" onClick={() => setPop(!pop) || setWho(false || setWhat(false) || setStrength(false))}>POP</button>
+          <button className="btnStrength" type="button" onClick={() => setStrength(!strength) || setWho(false || setWhat(false) || setPop(false))}>Mes Skills</button>
+          <button className="btnWhat" type="button" onClick={() => setWhat(!what) || setPop(false) || setStrength(false) || setWho(false)}>What else ?</button>
```

Comment:
weird use of ||. I guess what you're trying to do is:

    onClick={() => { setWho(!who); setStrength(false); setWhat(false); setPop(false) }}

consider having a unique state for every part:

    const allClosed = { who: false, strength: false, what: false, pop: false };

    function Screen() {
      const [isOpen, setOpen] = useState(allClosed);
      const open = (keyToOpen) => {
        setOpen({ ...allClosed, [keyToOpen]: true });
      };

      return (
        <button className="btnWho" type="button" onClick={() => open('who')}>Qui suis-je ?</button>
      );
    }